### DISCUSSION TITLE
Question about Address Sanitizer

### DISCUSSION THREAD

**Discussion by ArfyFR** [microsoft/vscode-cpptools#7220] [General]
Hi, ASAN seems to be available in MSVC, see
https://devblogs.microsoft.com/cppblog/address-sanitizer-for-msvc-now-generally-available/

Wondering what is/will be the status for vscode-cpptools ?

**Comment by sean-mcmanus** [comment]
That appears to be a compiler level tool. I don't know of any current plans to integrate it into VS Code. You could try asking someone on the VS team.

**Reply by ArfyFR** [reply]
Funny, I found this into vscode ;)

https://github.com/microsoft/vscode/issues/73875
(I think this is only applicable to C++, see https://github.com/Microsoft/vscode-cpptools/issues ) ⇒ then close the issue

That's why I asked here ;)

**Reply by sean-mcmanus** [reply]
By "VS team" I mean "Visual Studio 2019" and not the "VS Code" team. You could file a request for them to port their support to VS Code at https://developercommunity.visualstudio.com/report?space=62&entry=suggestion  ...you could also file a feature request on us (https://github.com/microsoft/vscode-cpptools/issues) although we probably wouldn't add that unless it got more upvotes.

**Comment by CsatiZoltan** [comment]
Opened issue #11896.
